app-ingest/src/main/scala/com/azavea/rf/JobDefinition.scala

```diff
@@ -0,0 +1,55 @@
+package com.azavea.rf
+
+import spray.json._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+
+import geotrellis.vector.Extent
+import geotrellis.vector.io._
+
+import java.net.URL
+
+/** Each TileSource, which sits the bottom of any JobDefinition (see below) is made of tiff urls and a target extent */
+case class TileSource(urls: Array[String], extent: Extent)
```

Comment:
I think there were a few other things we were thinking about including in the job definition as well:
 - band information (so we can maybe keep the RGB layers together in a multiband tile so as not to waste bandwidth)
 - zoom levels we want to import into

Not sure if there was more -- @echeipesh and @lossyrob probably have a good idea though